crates/hyperdrive-math/src/lib.rs

```diff
@@ -15,6 +17,7 @@ use rand::{
 };
 pub use short::*;
 pub use utils::*;
+use yield_space::get_spot_price;
```

Comment:
We might want to do a `pub use yield_space::*;` so that all of the yield space functions are publicly accessible.

Reply:
This would replace this line and the line below.